addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -500,9 +517,15 @@ var SlideUploadDialog = Dialog.extend({
         if (!preventOnchange) {
             var input = file.name;
             var inputVal = input.substr(0, input.lastIndexOf('.')) || input;
-            this._formSetFieldValue('name', inputVal);
+            if (certif === undefined || certif === false) {
```

Comment:
you can test the input target instead, so you don't need to add a param to the method and add a second event handler to call the first one.